src/commands/fizzbuzz.js

```diff
@@ -39,7 +39,7 @@ class FizzBuzz extends Aquarius.Command {
   }
 
   message(msg) {
-    const input = Aquarius.Triggers.messageTriggered(msg, new RegExp('^fizzbuzz ([\\d]+)$', 'i'));
+    const input = Aquarius.Triggers.messageTriggered(msg, /^fizzbuzz ([\\d]+)$/i);
```

Comment:
I haven't figured out a good way to setup a test framework for this yet haha. Most of the bugs come from discord.js API changes, and I'm not quite sure how to catch those